_posts/2020-10-24-spring-boot-specification.md

```diff
@@ -0,0 +1,372 @@
+---
+title: "Intro to Specifications"
+categories: [spring-boot]
+date: 2020-10-24 00:00:00 +1100
+modified: 2020-10-24 20:00:00 +1100
+author: mmr
+excerpt: "All you need to know about spring boot specification"
+image:
+  auto: 0084-search
+---
+
```

Comment:
Please add a short introductory sentence or two before the GitHub include. Something that introduces the problem in a sentence and how Specifications solve it.